rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.76.0"
+channel = "1.78.0"
```

Comment:
nitpick: could you limit this PR to exactly this line, and not the other changes?